Makefile

```diff
@@ -40,6 +46,16 @@ SCRIPTS = \
 	run-ocsp.sh \
 	watchdog-report.sh
 
+SHELL_CMD=bash
+define BUILD_CONTAINER
+FROM golang:1.9.1-alpine
```

Comment:
Whether here or below, any reason we are not going with 1.11 or even 1.12?

Reply:
We are currently standardized on 1.9.1 across all of EVE. I feel the next iteration should be us going to 1.12 while at the same time transitioning to go modules.

Reply:
Can we split it into 2 steps? Do 1.11 or 1.12 now (both support `go.mod`), and then we will do modules as a separate step? Why conflate risks..